plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -56,6 +56,7 @@
 import io.trino.plugin.hive.statistics.HiveStatisticsProvider;
 import io.trino.plugin.hive.util.HiveBucketing;
 import io.trino.plugin.hive.util.HiveUtil;
+import io.trino.plugin.hive.util.HiveWriteUtils;
```

Comment:
Please update `BaseS3AndGlueMetastoreTest`. Also, it would be better to increase the test coverage about other file systems, probably in `AbstractTestHiveFileSystem`. 

As you may know, we recently suffered from an issue about table locations having double slash.

Reply:
tests added

Reply:
@ebyhr can you please launch it with secrets for cloud tests